settings.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
     rootProject.name = 'tutteli-gradle'
-    gradle.ext.version = '0.27.3'
-    gradle.ext.previous_version = '0.26.1'
+    gradle.ext.version = '0.28.0'
```

Comment:
please keep the old version numbers, I'll increase them once I create a new release. 
You could change it to 028.0-SNAPSHOT if you like

Reply:
Done. I used 0.28.0-SNAPSHOT so we can test locally.